### ISSUE TITLE
Debian 11. Session doesn't inform about broken connection.

### ISSUE THREAD

**Issue by Nikolay-Po** [evgeniy1294/protodb#2]
If USB-UART converter was disconnected from a host, the session continues "successfully" sending Tx data. The problem may be discovered only after session stop. New start with disconnected serial device become impossible (which is correct).

**Comment by Nikolay-Po** [comment]
Some sort of informing about broken connection is needed. It was discouraging to search why I didn't see a signal on UART when the connection to host was physicaly absent.

**Comment by Nikolay-Po** [comment]
Sorry for my broken English, I don't have Russian locale at this machine yet.
Something like this were before session become sending data into a blackhole:

```
[ 5472.650722] ch341-uart ttyUSB1: usb_serial_generic_read_bulk_callback - urb stopped: -32
[ 5472.667102] usb 1-2: USB disconnect, device number 3
[ 5472.667498] usb 1-2: failed to send control message: -19
[ 5472.667735] ch341-uart ttyUSB1: ch341-uart converter now disconnected from ttyUSB1
[ 5472.667757] ch341 1-2:1.0: device disconnected

```

**Comment by evgeniy1294** [comment]
Closed since version 1.0.4
